web/themes/joinup/templates/rdf_entity/rdf-entity--about.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Themeing for rdf entities.
+ */
+#}
+
+
+<div{{ attributes }}>
```

Comment:
I decided to remove this template and remove the label in the preprocess function.